webapp/components/admin_console/storage_settings.jsx

```diff
@@ -113,6 +114,29 @@ export default class StorageSettings extends AdminSettings {
                     disabled={!this.state.enableFileAttachments}
                 />
             );
+
+            amazonSSEComp =
+                (
+                    <BooleanSetting
+                        id='amazonS3SSE'
+                        label={
+                            <FormattedMessage
+                                id='admin.image.AmazonSSETitle'
+                                defaultMessage='Enable Server-Side Encryption for Amazon S3:'
+                            />
+                        }
+                        placeholder={Utils.localizeMessage('admin.image.AmazonSSEExample', 'Ex "false"')}
+                        helpText={
+                            <FormattedHTMLMessage
+                                id='admin.image.AmazonSSEDescription'
```

Comment:
ugg...wait nvm.  I just realized the original PR didn't have them correctly.  I'll submit a fix soon.